### DISCUSSION TITLE
1.4.1 Clarification

### DISCUSSION THREAD

**Discussion by EmptyEnd6893** [w3c/wcag#3771] [Q&A] [answered]
For some time, I have been wondering about the following note for Use of Color:

> If content is conveyed through the use of colors that differ not only in their hue, but that also have a significant difference in lightness, then this counts as an additional visual distinction, as long as the difference in relative luminance between the colors leads to a contrast ratio of 3:1 or greater. For example, a light green and a dark red differ both by color (hue) and by lightness, so they would pass if the contrast ratio is at least 3:1. Similarly, if content is distinguished by inverting an element's foreground and background colors, this would pass (again, assuming that the foreground and background colors have a sufficient contrast).
> 
> However, if content relies on the user's ability to accurately perceive or differentiate a particular color an additional visual indicator will be required regardless of the contrast ratio between those colors. For example, knowing whether an outline is green for valid or red for invalid.

If I understand correctly, this means that lightness counts as a differentiating factor if content is 'conveyed through the use of colors', but not when content 'relies on the user's ability to accurately perceive or differentiate a particular color'. Could someone please clarify the distinction between these terms?

**Comment by patrickhlauke** [comment, marked as answer]
it's the difference between "the user needs to be able to just perceive that there's some difference between the two items" (for instance, to differentiate a focused link from a non-focused link), which is fine, versus "the user needs to be able to infer some characteristics about the item based purely on the color used" (for instance, user needing to specifically perceive red items that are errors compared to green items that are correct). in the latter case, a difference in lightness can help a user perceive that the two things are *different*, but not specifically what they actually represent. and in that case, you need something else there (like an "X" for errors, or similar visual differentiation ... or simply extra text that prefixes things as "Error: ..." when they're errors, etc)

**Reply by EmptyEnd6893** [reply]
Thanks for your reply.
